pegasus/sites.v3/code.org/views/homepage_gallery.haml

```diff
@@ -4,7 +4,7 @@
 #gallerycolumn.same-height-column
   %a.nounderscorehover.linktag#projects-images{href: CDO.studio_url("projects/public")}
     .headingbox{style: "background-color: rgb(#{bgc_body})"}
-      = I18n.t('homepage_gallery_header', project_count: (round_project_count_to_million[fetch_metrics['project_count']])
+      = I18n.t('homepage_gallery_header', project_count: fetch_rounded_project_count_to_million)
```

Comment:
I think this could read well to be `project_count: fetch_project_count['rounded_millions']`.